doc/source/schemas/alleleAnnotationmethods.rst

```diff
@@ -0,0 +1,700 @@
+AlleleAnnotationMethods
+***********************
+
+ .. function:: searchVariantAnnotationSets(request)
+
+  :param request: SearchVariantAnnotationSetsRequest: This request maps to the body of `POST /variantannotationsets/search` as JSON
+  :return type: SearchVariantAnnotationSetsResponse
+  :throws: GAException
+
+Returns a list of available variant annotation sets
+`POST /variantannotationsets/search` must accept a JSON version of
+`SearchVariantAnnotationSetsRequest` as the post body and will return a JSON
+version of `SearchVariantAnnotationSetsResponse`.
+
+ .. function:: searchVariantAnnotations(request)
+
+  :param request: SearchVariantAnnotationsRequest: This request maps to the body of `POST /variantannotations/search` as JSON.
+  :return type: SearchVariantAnnotationsResponse
+  :throws: GAException
+
+Gets a list of `VariantAnnotations` matching the search criteria.
+
+`POST /variantannotations/search` must accept a JSON version of
+`SearchVariantAnnotationsRequest` as the post body and will return a
+JSON version of `SearchVariantAnnotationsResponse`.
+
+ .. function:: getVariantAnnotationSet(id)
+
+  :param id: string: The ID of the `VariantAnnotationSet`.
+  :return type: org.ga4gh.models.VariantAnnotationSet
+  :throws: GAException
+
+Gets an `VariantAnnotationSet` by ID.
+`GET /variantannotationsets/{id}` will return a JSON version of
+`VariantAnnotationSet`.
+
+.. avro:error:: GAException
+
+  A general exception type.
+
+.. avro:enum:: Strand
+
+  :symbols: NEG_STRAND|POS_STRAND
+  Indicates the DNA strand associate for some data item.
+  * `NEG_STRAND`: The negative (-) strand.
+  * `POS_STRAND`:  The postive (+) strand.
+
+.. avro:record:: Position
```

Comment:
There seems to be quite a lot of repetition here: shouldn't Strand, Position, etc already be defined elsewhere?